vector/src/main/res/values/strings.xml

```diff
@@ -3473,6 +3476,13 @@
 
     <string name="create_space_identity_server_info_none">You are not currently using an identity server. In order to invite teammates and be discoverable by them, configure one below.</string>
 
+
+    <string name="finish_setting_up_discovery">Finish setting up discovery.</string>
+    <string name="discovery_invite">Invite by email, find contacts and more…</string>
+    <string name="finish_setup">Finish setup</string>
+    <string name="discovery_section">Discovery (%s)</string>
```

Comment:
Please add a comment to tell the translators by what will be replaced  `%s` at runtime

Reply:
done